### DISCUSSION TITLE
Unable to unstall CU22 on SQL 2019

### DISCUSSION THREAD

**Discussion by vitalic023** [dataplat/dbatools#9123] [General]
He there, 
When installing CU22(KB5027702) using dbatools, I get the warning:
_[Get-SqlInstanceUpdate] 15.0.4312 on computer [computername] is already the
latest available._
The 15.0.4312 is CU20(kb5024276), its current installed version and its not the latest, obviously.
Code used:
_$splatUpdateInst = @{
ComputerName = "computername"
Path = 'C:\temp\SQLServer2019-KB5027702-x64.exe' 
}
Update-DbaInstance @splatUpdateInst_

Any help suggestion will be greatly appreciated.

VJ

**Comment by niphlod** [comment]
hi @vitalic023 , do you have an updated index ? run `Update-DbaBuildReference` , then retry the `Update-DbaInstance`.

long story: every dbatools ships with a file which has the sql server builds AT THE TIME that dbatools release is shipped. Via `Update-DbaBuildReference` that file is fetched from internet with the sql server builds AT THE TIME that you run the command.
`Update-DbaInstance` reason on the "applyability" of a CU on the same reference info, so, if it's not updated, it doesn't know there is, in fact, a newer CU to install.

**Comment by vitalic023** [comment]
Thanks Simone, that fixed it 😊

Kindest regards,


Vitali
